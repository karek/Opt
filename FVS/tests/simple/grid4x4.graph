a b
b c
c d
p q
q r
r s
x y
y z
z w
q p
A B
B C
C D
a p
p x
x A
b q
q y
y B
c r
r z
z C
d s
s w
w D
