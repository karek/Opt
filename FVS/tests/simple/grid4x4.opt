4
p r y z 
