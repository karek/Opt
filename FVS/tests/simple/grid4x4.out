4
q s z B 
